docs/devel/storage.md

```diff
@@ -1,3 +1,104 @@
-# Storage
+# Getting started with storage
 
-TODO
+This document assumes you have a working [development environment](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md).
+
+
+**We are using hostPath PVs, which are NOT suitable for a production environment, this document assumes you are setting up a SINGLE node development environment.**
+
+In order to use disk images one must configure an appropriate storage. In this document we will be using hostPath Persistent Volumes (PVs) and making Persistent Volume Claims against that PV. If you have some other mechanism to get PVCs you can skip the 'deploy hostPath PVC provisioner' section of this document. Throughout the entire document, I am assuming we are deploying into the 'default' namespace. You can also deploy into a different name space if you want.
+
+## Create hostpath directory on node.
+The hostPath provisioner needs to write to a directory on the node, we will need to create this directory before installing the hostPath provisioner
+
+```bash
+cluster/cli.sh ssh node01 -- sudo mkdir /var/run/kubevirt/hostpath
+
+## deploy hostPath PVC provisioner
+
+We are using the hostPath provisioner from [here](https://github.com/MaZderMind/hostpath-provisioner). Simply run from :
+
+```bash
```

Comment:
You missed the closing of the syntax section above, so this gets rendered in the markup as is here.  I'm assuming you wanted to close the other code block out at L#17 and start a new one here?